thriftpy2/thrift.py

```diff
@@ -13,12 +13,22 @@
 import linecache
 import types
 
-from ._compat import with_metaclass
+from ._compat import with_metaclass, PY3
+if PY3:
+    from itertools import zip_longest
+else:
+    from itertools import izip_longest as zip_longest
 
 
-def args2kwargs(thrift_spec, *args):
-    arg_names = [item[1][1] for item in sorted(thrift_spec.items())]
-    return dict(zip(arg_names, args))
+def args2kwargs(thrift_spec, *args, **kwargs):
```

Comment:
`args_to_kwargs` is better than `args2kwargs` I think.

Reply:
Done